prac_07/guitar.py

```diff
@@ -0,0 +1,23 @@
+
+CURRENT_YEAR = 2017
+VINTAGE_AGE = 50
+
+
+class Guitar:
+
+    def __init__(self, name="", year=0, cost=0):
```

Comment:
do you really need the predefined values if you have already changed each value to the needed one in myguitars program